requirements.txt

```diff
@@ -2,10 +2,11 @@
 colour-runner==0.0.4
 coverage==4.2
 dj-database-url==0.4.1
-django>=1.8,<1.9
+django>=1.10,<1.11
```

Comment:
Packaging is not my strong point but:  isn't this used for the package once it hits pypi? Should we be moving it into the setup.py to exclude the test reqs?